src/lando/main/scm/__init__.py

```diff
@@ -5,7 +5,10 @@
 from lando.main.scm.git import *
 from lando.main.scm.hg import *
 
+# These can only be determined when all the subclasses of the AbstractSCM have been defined.
+SCM_CHOICES = {
+    klass.scm_type(): klass.scm_name() for klass in AbstractSCM.__subclasses__()
+}
 SCM_IMPLEMENTATIONS = {
-    SCM_GIT: GitSCM,
-    SCM_HG: HgSCM,
+    klass.scm_type(): klass for klass in AbstractSCM.__subclasses__()
```

Comment:
@cgsheeh Here's my suggestion to address your comment in https://github.com/mozilla-conduit/lando/pull/154#discussion_r1866870640